documentation/chatmeleon-swagger.yaml

```diff
@@ -283,6 +284,208 @@ paths:
                   type: string
                   format: uri
 
+  /friend/send:
+    post:
+      description: Create a friend sending and pending relationship between two users.
+      tags:
+        - relationships
+      summary: Create a relationship.
+      security:
+        - bearerAuth: []
+      responses:
+        "200":
+          description: Relationship created successfully.
+        "401":
+          description: Unauthorized - JWT token is missing or invalid.
+        "403":
+          description: Unauthorized - User is unauthorized.
+
+      requestBody:
+        description: The ID of the other user to create relationship.
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                relatedUserId:
+                  type: string
+                  pattern: "^[0-9a-f]{24}$"
+
+  /friend/accept:
+    put:
```

Comment:
I feel like the POST method would better represent "submitting acceptance to the friend request". Using PUT to "update the status of the friend request" seems a little weird to me. It should be more like closing the friend request, not updating it.

It's fine if you want to keep it as is though

Reply:
yea tai no chi update cai type cua relationship 2 dua chu ko phai tao cai moi xong xoa cai cu a